src/services.js

```diff
@@ -63,17 +64,27 @@ const services = {
     })
   }, svc)),
   /**
-   * Kills all running services with detached process
+   * Kills all running, non-persisted services
+   * @returns {object} promise
    */
-  stop: () => _.unless(
-    _.isEmpty,
-    _.pipe([
-      _.filter(_.test(/dl_/)),
-      _.map(svc => `docker stop ${svc} && docker rm ${svc}`),
-      _.join(' && '),
-      proc.runDetached
-    ])
-  )(services.running)
+  stop: () => {
+    const errors = []
+    return Promise.all(_.unless(
```

Comment:
I believe you can skip the `isEmpty` check, since if it's empty there will be nothing to filter/map over and thus it will resolve immediately.

Probably either consider that extraneous check or just return early if it's empty for cleanliness?